tests/alive-tv/vector/x86/sse2_psrl_w-poison1.srctgt.ll

```diff
@@ -0,0 +1,10 @@
+define <8 x i16> @src(<8 x i16> %v) {
+  %1 = call <8 x i16> @llvm.x86.sse2.psrl.w(<8 x i16> %v, <8 x i16> <i16 0, i16 0, i16 poison, i16 0, i16 0, i16 0, i16 0, i16 0>)
+  ret <8 x i16> %1
+}
+
+define <8 x i16> @tgt(<8 x i16> %v) {
```

Comment:
this sort of semantics is not common in LLVM. We've been using element-wise poison semantics. Is there a reason to deviate from that?

Reply:
The instruction uses the second vector operand as a 64 bit integer, whether the result is poison is not dependent on the element higher than 64 bit, am I right?